opentelemetry-sdk/src/opentelemetry/sdk/_metrics/metric_exporter.py

```diff
@@ -0,0 +1,17 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
```

Comment:
ConsoleMetricsExporter?

Reply:
`ConsoleMetricsExporter` and `SDKMetricReader` were defined before in the code. This PR removes them from `__init__.py` in the sdk so they need to be redefined somewhere else.

Reply:
Ok, but that would be better added in another PR, this one does not use those symbols.